jvm/ml4ir-inference/src/main/java/ml4ir/inference/tensorflow/SimpleSequenceExampleExecutor.java

```diff
@@ -0,0 +1,35 @@
+package ml4ir.inference.tensorflow;
+
+import com.google.common.collect.ImmutableMap;
+import ml4ir.inference.tensorflow.data.ModelFeaturesConfig;
+import ml4ir.inference.tensorflow.data.StringMapSequenceExampleBuilder;
+import org.tensorflow.example.SequenceExample;
+
+import java.util.List;
+import java.util.Map;
+
+public class SimpleSequenceExampleExecutor {
+    private final SequenceExampleExecutor executor;
+    private final StringMapSequenceExampleBuilder protoBuilder;
+    public SimpleSequenceExampleExecutor(String modelBundlePath,
+                                         String inputNodeName,
+                                         String outputNodeName,
+                                         String featureConfigPath) {
+        executor = new SequenceExampleExecutor(
+                modelBundlePath,
+                ModelExecutorConfig.apply(inputNodeName, outputNodeName)
+        );
+        protoBuilder = StringMapSequenceExampleBuilder.withFeatureProcessors(
+                ModelFeaturesConfig.load(featureConfigPath),
+                ImmutableMap.of(),
+                ImmutableMap.of(),
+                ImmutableMap.of()
+        );
+    }
+
+    public float[] predict(Map<String, String> contextFeatures, List<Map<String, String>> recordFeatures) {
```

Comment:
Could we add a comment about what should be the keys/values of recordFeatures and recordFeatures? (with an example?)